src/util/MongoBinaryDownload.ts

```diff
@@ -108,9 +116,9 @@ export default class MongoBinaryDownload {
     return mongoDBArchive;
   }
 
-  async makeMD5check(urlForReferenceMD5: string, mongoDBArchive: string): Promise<?boolean> {
+  async makeMD5check(urlForReferenceMD5: string, mongoDBArchive: string): Promise<boolean> {
     if (!this.checkMD5) {
-      return undefined;
+      return false; // TODO : why was it return undefined; ?
```

Comment:
@nodkz Changed this return to `return false` if it's ok for you

Reply:
Yep it should return `boolean | void` 

We don't know exactly will be passed md5 checks or not, when it disabled via configs.
So for such case, we return `undefined` because the check was not made. 😈

Reply:
@nodkz Ok I changed back to return undefined.